extensions/amp-bind/amp-bind.md

```diff
@@ -76,48 +141,76 @@ The state can be initialized with the `amp-state` component:
+| Size of [AMP elements](https://www.ampproject.org/docs/reference/components) | `[width]`<br>`[height]` | Changes the width and/or height of the AMP element. |
+| Element-specific attribute | [Various](#element-specific-attributes). | Boolean expression results toggle boolean attributes.
 
-- For security reasons, binding to `innerHTML` is disallowed
-- All attribute bindings are sanitized for unsafe URL protocols (e.g. `javascript:`)
+- For security reasons, binding to `innerHTML` is disallowed.
+- All attribute bindings are sanitized for unsafe values, e.g. URL protocols (e.g. `javascript:`).
 
 #### Element-specific attributes
 
-Most attributes accepted by the [AMP Validator](https://validator.ampproject.org/) for AMP and non-AMP elements, are bindable.
+Only binding to the following components and attributes are allowed. Most bindable attributes correspond to a non-bindable counterpart, e.g. for `<amp-img>`, `[src]` changes the value of `src`.
 
-There are also special bindable attributes without non-bindable counterparts:
+(*) Denotes bindable attributes that don't have a non-bindable counterpart.
 
-| Component | Attribute | Details | Example |
-| --- | --- | --- | --- |
-| amp-carousel[type=slides] | `[slide]` | The currently displayed slide index. | [Linked carousels](https://ampbyexample.com/advanced/image_galleries_with_amp-carousel/#linking-carousels-with-amp-bind)
-| amp-selector | `[selected]` | The `option` attribute values of the currently selected children elements. | [Linked carousels](https://ampbyexample.com/advanced/image_galleries_with_amp-carousel/#linking-carousels-with-amp-bind)
+| Component | Attribute(s) | Behavior |
+| --- | --- | --- |
+| `<amp-brightcove>` | `[data-account]`<br>`[data-embed]`<br>`[data-player]`<br>`[data-player-id]`<br>`[data-playlist-id]`<br>`[data-video-id]` | Changes the displayed Brightcove video. |
+| `<amp-carousel type=slides>` | `[slide]`* | Changes the currently displayed slide index. [See an example](https://ampbyexample.com/advanced/image_galleries_with_amp-carousel/#linking-carousels-with-amp-bind).
+| `<amp-iframe>` | `[src]` | Changes the iframe's source URL. |
+| `<amp-img>` | `[alt]`<br>`[attribution]`<br>`[src]`<br>`[srcset]` | Various. See [amp-img attributes](https://www.ampproject.org/docs/reference/components/media/amp-img#attributes). |
+| `<amp-selector>` | `[selected]`* | Changes the currently selected children element(s)<br>identified by their `option` attribute values. Supports a comma-separated list of values for multiple selection. [See an example](https://ampbyexample.com/advanced/image_galleries_with_amp-carousel/#linking-carousels-with-amp-bind).
+| `<amp-video>` | `[alt]`<br>`[attribution]`<br>`[controls]`<br>`[loop]`<br>`[poster]`<br>`[preload]`<br>`[src]` | Various. See [amp-video attributes](https://www.ampproject.org/docs/reference/components/media/amp-video#attributes). |
```

Comment:
Suggestion: here and elsewhere, how would you feel about replacing 


"Various. See ${ELEMENT} attributes"

with 

"See corresponding ${ELEMENT} attributes"